test/integration/protocol_integration_test.cc

```diff
@@ -1355,16 +1363,20 @@ TEST_P(ProtocolIntegrationTest, ConnDurationInflightRequest) {
   test_server_->waitForCounterGe("cluster.cluster_0.upstream_cx_total", 1);
   test_server_->waitForCounterGe("cluster.cluster_0.upstream_rq_200", 1);
 
-  ASSERT_TRUE(codec_client_->waitForDisconnect());
+  ASSERT_TRUE(codec_client_->waitForDisconnect(std::chrono::milliseconds(10000)));
```

Comment:
This is probably OK, but if we see future flakes, this should be increase by some small fudge factor.